raft-ann-bench/gpu/Dockerfile

```diff
@@ -12,24 +12,31 @@ ARG RAPIDS_VER
 
 COPY condarc /opt/conda/.condarc
 
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+
 # Create a data folder accessible by any user so mounted volumes under it can be accessible
 # when the user passes their uid to docker run with -u $(id -u)
 # Also add the conda_prefix config to the global bashrc file so that all users have it correctly configured.
-RUN mkdir /data && chmod 777 /data \
-      && ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh; \
-      echo ". /opt/conda/etc/profile.d/conda.sh; conda activate base" >> /etc/bash.bashrc
+RUN <<EOF
+mkdir /data
+chmod 777 /data
+ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh
```

Comment:
```suggestion
```

Since this line is now throwing an error message, we can remove it since it's set in the parent image below:

- https://github.com/rapidsai/miniforge-cuda/blob/65db643ee50f34494cd68f962f0f9c5bf90eda6b/Dockerfile#L36